waspc/src/Wasp/Generator/JsImport.hs

```diff
@@ -15,19 +15,19 @@ import Wasp.JsImport
     makeJsImport,
   )
 
-type PathFromImportLocationToExtCodeDir = Path Posix (Rel GeneratedExternalCodeDir) Dir'
+type PathFromImportLocationToSrcDir d = Path Posix (Rel ()) (Dir d)
 
 extImportToJsImport ::
   GeneratedSrcDir d =>
   Path Posix (Rel d) (Dir GeneratedExternalCodeDir) ->
-  Path Posix (Rel GeneratedExternalCodeDir) Dir' ->
+  PathFromImportLocationToSrcDir d ->
   EI.ExtImport ->
   JsImport
```

Comment:
Just a quick idea, no need to do it but wanted to offer it in case you like it:
```hs
type ImportLocation = ()

extImportToJsImport ::
  GeneratedSrcDir srcDir =>
  Path Posix (Rel srcDir) (Dir GeneratedExternalCodeDir) ->
  Path Posix (Rel ImportLocation) (Dir srcDir)
  EI.ExtImport ->
  JsImport
```

it gives some uniformity to types used in this signature (it is a bit weird when one is behind type alias and another on is not) but still captures the info that we are dealing with import location here.

What was also a bit weird with the old `type` was that it says it goes to SrcDir but doesn't enforce constraint on its own so you could use it somewhere and not put constraint on it.

Reply:
I'll address this suggestion in a separate PR 👍